.github/workflows/test.yml

```
@@ -69,13 +69,10 @@ jobs:
       fail-fast: false
       matrix:
         python-version: ["3.9", "3.10", "3.11"]
-        hoomd-version: ["2.9.7", "3.11.0", "4.7.0"]
+        hoomd-version: ["3.11.0", "4.7.0"]
```

Bump this to 4.9.1 while we're modifying our test list:
```suggestion
        hoomd-version: ["3.11.0", "4.9.1"]
```